code/modules/admin/smites/rod.dm

```diff
@@ -1,11 +1,16 @@
 /// Throw an immovable rod at the target
 /datum/smite/rod
 	name = "Immovable Rod"
+	var/force_looping = FALSE
 
 /datum/smite/rod/effect(client/user, mob/living/target)
 	. = ..()
 	var/turf/target_turf = get_turf(target)
 	var/startside = pick(GLOB.cardinals)
 	var/turf/start_turf = spaceDebrisStartLoc(startside, target_turf.z)
 	var/turf/end_turf = spaceDebrisFinishLoc(startside, target_turf.z)
-	new /obj/effect/immovablerod(start_turf, end_turf, target)
+	new /obj/effect/immovablerod(start_turf, end_turf, target, force_looping)
+
+/datum/smite/rod/loopy
```

Comment:
I'd rather this just be in the `configure` for the smite. Think the :b: line.

Reply:
Good call. I'll get this done in tomorrow. Or today. Depending on your timezone.

Reply:
9:25 pm GOGOGO